src/libraries/Common/tests/Tests/Interop/OSReleaseTests.cs

```diff
@@ -50,9 +50,38 @@ public void GetPrettyName_NoFile_ReturnsNull()
         [Fact, PlatformSpecific(TestPlatforms.Linux)]
         public void GetPrettyName_CannotRead_ReturnsNull()
         {
+            // This test only applies for non-root users.
+            if (Environment.IsPrivilegedProcess)
+            {
+                return;
+            }
+
             string path = CreateTestFile();
             File.SetUnixFileMode(path, UnixFileMode.None);
-            Assert.ThrowsAny<Exception>(() => File.ReadAllText(path));
+
+            Assert.ThrowsAny<Exception>(() => File.ReadAllText(path)); 
+
+            string? name = Interop.OSReleaseFile.GetPrettyName(path);
+            Assert.Null(name);
+        }
+
+        [Fact, PlatformSpecific(TestPlatforms.Linux)]
+        public void GetPrettyName_NonePrivileges_CanRead_ReturnsNull()
+        {
+            // This test only applies for root users.
+            if (!Environment.IsPrivilegedProcess)
```

Comment:
Perhaps put `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsPrivilegedProcess))]` on the test signatures? (I think it works with `Environment` too).